src/test/java/cucumber/steps/LegalEntitySteps2.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package cucumber.steps;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import java.util.Map;
+import org.apache.log4j.LogManager;
+import org.apache.log4j.Logger;
+import org.testng.asserts.SoftAssert;
+import salesforce.entities.LegalEntity;
+import salesforce.ui.pages.AppPageFactory;
+import salesforce.ui.pages.legalentity.LegalEntityPageAbstract;
+import salesforce.ui.pages.legalentity.NewLegalEntityPageAbstract;
+import static salesforce.config.EnvironmentConfig.getSalesforceVersion;
+
+public class LegalEntitySteps2 {
```

Comment:
Change the class name to just "LegalEntitySteps"

Reply:
already changed